dataplane/linux/vxlan_mgr.go

```diff
@@ -175,10 +179,12 @@ func (m *vxlanManager) CompleteDeferredWork() error {
 
 	if m.routesDirty {
 		// Iterate through all of our L3 routes and send them through to the route table.
-		var routes []routetable.Target
+		var vxlanRoutes []routetable.Target
+		var defaultRoutes []routetable.Target
 		for _, r := range m.routesByDest {
 			logCtx := logrus.WithField("route", r)
 			cidr, err := ip.CIDRFromString(r.Dst)
+
```

Comment:
Here's another blank line we should get rid of. In general it's nice to keep the handling of the error within a block with the call that generated the error.